nativelib/src/main/resources/scala-native/time_millis.c

```diff
@@ -11,28 +13,32 @@ long long scalanative_current_time_millis() {
 
 #if defined(_WIN32)
     // January 1, 1970 (start of Unix epoch) in "ticks"
-    long long UNIX_TIME_START = 0x019DB1DED53E8000;
-    long long TICKS_PER_MILLIS = 10000; // a tick is 100ns
+#define UNIX_TIME_START 0x019DB1DED53E8000LL
+#define MILLIS_PER_NANO 1000000
```

Comment:
Sorry, my problem, I am confused by the name MILLIS_PER_NANO.  The obvious answer is Integer zero (or floating 1/(1000 * 1000)).  Did you mean NANOS_PER_MILLI,
as used in the "else" path below?  If yes, could the constants
be unified and moved above the "if/else"? Single Point of Truth.